docs/dev/reference/config.md

```diff
@@ -251,7 +252,47 @@ website over `http`, note that the cookie name will be `csrf_http-contao_csrf_to
 However, protecting your users from CSRF attacks but let them submit the form via unsecured `http` connections is
 not really a valid use case. 
 {{% /notice %}}
-    
+
+### `COOKIE_REMOVE_FROM_DENY_LIST`
+
+{{< version "4.10" >}}
+
+In case you don't want to manage the whole `COOKIE_ALLOW_LIST` because you're unsure what your application needs but
+you want to disable one of the existing entries on the deny list that's managed by Contao, you can specify this using:
+
+```
+COOKIE_REMOVE_FROM_DENY_LIST=__utm.+
```

Comment:
Is this env var just a comma separated list? If yes, may be show that in the example :)

Reply:
Done